src/Mobs/Wolf.cpp

```diff
@@ -18,7 +18,7 @@ cWolf::cWolf(void) :
 	m_IsBegging(false),
 	m_IsAngry(false),
 	m_OwnerName(""),
-	m_CollarColor(14)
+	m_CollarColor(E_META_DYE_ORANGE)
```

Comment:
According to the changelog on http://minecraft.gamepedia.com/Wolf they were red, then orange, then red again in 1.9 - it's a mess really.